api/resource/definitions/network/network.proto

```diff
@@ -0,0 +1,300 @@
+syntax = "proto3";
+
+package network;
```

Comment:
I think we need longer fully-qualified package names: https://developers.google.com/protocol-buffers/docs/style

e.g. `package talos.resource.definitions.network`

Reply:
Done!